src/index.html

```diff
@@ -5,10 +5,31 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="header">
+      <div class="header__nav">
+        <a href="#home" class="logo"><img src="../src/images/logo.png" alt="MOYO"></a>
+        <nav class="nav">
+          <ul class="nav__list">
+            <a href="#apple" class="nav__link is-active nav__link--gap">APPLE</a>
+            <a href="#samsung" class="nav__link nav__link--gap">SAMSUNG</a>
+            <a href="#smartphones" class="nav__link nav__link--gap">SMARTPHONES</a>
+            <a href="#laptops-computers" data-qa="hover"
+                class="nav__link nav__link--gap">LAPTOPS & COMPUTERS</a>
+            <a href="#gadgets" class="nav__link nav__link--gap">GADGETS</a>
+            <a href="#tablets" class="nav__link nav__link--gap">TABLETS</a>
+            <a href="#photo" class="nav__link nav__link--gap">PHOTO</a>
+            <a href="#video" class="nav__link nav__link--gap">VIDEO</a>
+          </ul>
```

Comment:
The tag `ul` must contain a list element `li`
```suggestion
          <ul class="nav__list">
            <li class="nav__item">
              <a href="#apple" class="nav__link is-active">APPLE</a>
            </li>
            <a href="#samsung" class="nav__link nav__link--gap">SAMSUNG</a>
            <a href="#smartphones" class="nav__link nav__link--gap">SMARTPHONES</a>
            <a href="#laptops-computers" data-qa="hover"
                class="nav__link nav__link--gap">LAPTOPS & COMPUTERS</a>
            <a href="#gadgets" class="nav__link nav__link--gap">GADGETS</a>
            <a href="#tablets" class="nav__link nav__link--gap">TABLETS</a>
            <a href="#photo" class="nav__link nav__link--gap">PHOTO</a>
            <a href="#video" class="nav__link nav__link--gap">VIDEO</a>
          </ul>
```